### DISCUSSION TITLE
Upgraded all my sites to Oqtane 5.2.0 hosted on SmarterASP.NET

### DISCUSSION THREAD

**Discussion by vineet-medical** [oqtane/oqtane.framework#4544] [Show and tell]
I just upgraded https://learnoqtane.com/ to Oqtane 5.2.0 from 5.1.2 which is hosted on [SmarterASP.NET](https://www.smarterasp.net/index?r=vnetonline)

![image](https://github.com/user-attachments/assets/4ac53015-43aa-4bae-bec3-b0a49b99ee47)

**Comment by W6HBR** [comment]
Can you comment on what SmarterASP.net plan you have and how the performance is on that plan?  Also, if you started on a lower plan and found performance issues and then moved up - would be helpful to know.  Thansk.

**Reply by leigh-pointer** [reply]
Consider also looking into interserver.net
.net core/asp.net windows hosting unlimited everything through Plesk interface.
Data
Domains
DNS entries 
Email accounts
Databases
Disk space
All for $8 a month 

Seconds to set up Oqtane

**Comment by GregStevenson** [comment]
Hi Jon,

I have two premium accounts with SmarterASP.net: one personal account with lots of domains and one for hosting a startup that will require a lot of disk space/file storage. I started with the trial but quickly ran into its limitations. I believe my needs are a special case, so most people can use the trial without issue. Here is a [link](https://www.smarterasp.net/hosting_plans) comparing the various plans. The list is extensive but not exhaustive. For example, I also needed non-standard outgoing ports, which require the premium plan or above. Again, this is not a typical requirement.

In my opinion, their customer service via chat is excellent and available 24/7. They list their phone number and address, but using chat and service tickets seems to get the job done very efficiently.

I also like that they are based in Southern California for performance reasons since I need to transfer a lot of data to one server for testing. I'm near Irvine, and I think you're in Huntington Beach. Vineet is on the other side of the globe, so he may be a better data point for performance. I did not select Premium for performance-related reasons.

Hope this helps,

**Reply by ADefWebserver** [reply]
Yeah that *is* inexpensive compared to Azure.

**Comment by vnetonline** [comment]
Interesting you say that @ADefWebserver I find Azure much more expensive. 

On a monthly basis, Azure costs me ~ $155 AUD

![image](https://github.com/user-attachments/assets/3970bdfa-f7d1-4015-9fb6-a6551037309f)

Here is my split

- $102.693 AUD per month Basic AppPlan
- $48.46 AUD SQL Server on Standard Plan

Total: 151.153

![image](https://github.com/user-attachments/assets/c0ade779-63f3-44de-b10e-4ba6e5471e57)

I have moved all my sites to SmarterASP.NET, which has outstanding customer support and excellent performance. In addition, they are prompt when installing the latest .NET 8 Runtimes; I just got them to install .NET 8.0.8

The cost is $7.95 USD per month compared to ~$155 AUD per month. I have 32 sites installed on one instance, and things work great. 

Anyone looking to host your Oqtane sites with outstanding support and excellent performance

![image](https://github.com/user-attachments/assets/fa36059f-5254-454f-bc98-6b362458751c)

[Host my SIte on Oqtane with SmarterASP.Net](https://www.smarterasp.net/index?r=vnetonline)

**Reply by leigh-pointer** [reply]
@vineet-medical, it seems you’ve misunderstood @ADefWebserver’s comment. He mentioned ‘inexpensive,’ indicating that it is more affordable compared to Azure.

**Comment by W6HBR** [comment]
@vineet-medical You said you are running 32 sites on one instance.  How many actual Oqtane instances is that?  Is it just one Oqtane instance with multiple tenant sites or is it multiple, separate instances?  All that for $7.95/mo?
